modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -0,0 +1,92 @@
+= Troubleshooting a Failing DataWeave Script
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+:page-aliases: dataweave-troubleshooting.adoc
+
+When you troubleshoot a failing script, one challenge to reproducing an error is having the same inputs the script had
+when it executed, especially in production environments where inputs can
+change unexpectedly.
+
+== DataWeave Exceptions
+
+When a function is called with the incorrect kind of argument, it throws the
+exception, `org.mule.weave.v2.exception.UnsupportedTypeCoercionException`.
+Causes of this exception include:
+
+* <<null_arg_not_accepted>>
+* <<mimetype_unset>>
+
+[[null_arg_not_accepted]]
+=== Function Does Not Accept Null Argument
+
+The most common cause of this exception occurs when one of the arguments is
+`Null` and the function does not accept `Null` as an argument. This issue
+results in an error message similar to the following:
+
+[source,weave,linenums]
+----
+You called the function '++' with these arguments:
+  1: Null (null)
+  2: String (" A text")
+
+But it expects one of these combinations:
+  (Array, Array)
+  (Date, Time)
+  (Date, LocalTime)
+  (Date, TimeZone)
+  (LocalDateTime, TimeZone)
+  (LocalTime, Date)
+  (LocalTime, TimeZone)
+  (Object, Object)
+  (String, String)
+  (Time, Date)
+  (TimeZone, LocalDateTime)
+  (TimeZone, Date)
+  (TimeZone, LocalTime)
+
+4| payload.message ++ " A text"
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Trace:
+  at main::++ (line: 4, column: 1)
+  at main::main (line: 4, column: 17)
+----
+
+One way to resolve this issue uses the `default` operator. For example,
```

Comment:
One way to resolve this issue uses the `default` operator >
I think it is clearer this way: One way to resolve this issue **is using** the `default` operator